ivy/data_classes/array/experimental/manipulation.py

```diff
@@ -173,6 +173,11 @@ def vstack(
             ivy.array([[1, 2],
                        [5, 6],
                        [7, 8]])
+
+                    x = ivy.array([1, 2])
+                    y = [ivy.array([[5, 6]]), ivy.array([[7, 8]])]
+                    ivy.vstack((x, y))
+                    ivy.vstack((x, y, x, y))
         """
```

Comment:
Hi @Aryan8912, I think this docstring edit and the edit in the `rst` file are not related to the PR. We generally want to keep each PR focused on a specific issue. Could you remove those?